apps/src/applab/api.js

```diff
@@ -380,8 +380,8 @@ export function startWebRequest(url, func) {
                            'func': func });
 }
 
-export function startWebRequestSync(url, func) {
-  return Applab.executeCmd(null, 'startWebRequestSync', {url, func});
+export function startWebRequestSync(url, func, headers) {
```

Comment:
Have you tested this out? I think `func` might need to be last in order for native-is-async to work.

Reply:
Great catch Josh! This was in fact broken and is fixed now.

I initially had it working with `func` last, swapped the param order, and then must not have properly retested.